config/locales/forms/cease_or_revoke_forms/en.yml

```
@@ -0,0 +1,35 @@
+en:
+  waste_carriers_engine:
+    cease_or_revoke_forms:
+      new:
+        heading: "Revoke or cease registration %{reg_identifier}"
+        tier:
+          upper: "Upper tier"
+          lower: "Lower tier"
+        registration_type:
+          carrier_dealer: "Carrier and dealer"
+          broker_dealer: "Broker and dealer"
+          carrier_broker_dealer: "Carrier, broker and dealer"
+        reason:
+          label: "Reason"
+          help_text: "Maximum 500 characters"
+        cease_or_revoke:
+          legend: "What do you want to do?"
+          options:
+            revoke: "Revoke"
+            cease: "De-register"
```

I noted it in the screenshot attached to the PR. All the error messaging refers to "cease" yet we have labelled the radio button "De-register". I know that's what it is currently called, but we're aiming to bring the terminology inline with Waste Exemptions.

```suggestion
            cease: "Cease"
```

---

Oh!! I thought from our last conversation that we need to keep `de-register` instead of `cease` for now

---

My bad. Just meant behind the scenes everything should work the same. But in the UI it's "cease" all the way.